content/tutorials/serving/modular-max-qwen.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Serve your LLM with MAX Serve
+weight: 2
+variants: -flyte +serverless +byoc +selfmanaged
+layout: py_example
+example_file: /external/unionai-examples/tutorials/max_serve/max_serve.py
```

Comment:
The diff shown below for the unionai-examples submodule doesn't show the addition of this file... it's possible I'm just looking at this wrong